go-owasm/api/lib.go

```diff
@@ -58,3 +60,12 @@ func run(code []byte, isPrepare bool, env EnvInterface) int32 {
 		},
 	}))
 }
+
+func WatToWasm(code []byte) ([]byte, error) {
+	inputSpan := copySpan(code)
+	defer freeSpan(inputSpan)
+	outputSpan := newSpan(1 * 1024 * 1024)
```

Comment:
Eventually we will need to take this as parameter or define it as a constant somewhere.